src/prefect/server/events/triggers.py

```diff
@@ -513,6 +514,13 @@ async def reactive_evaluation(event: ReceivedEvent, depth: int = 0):
                     await session.commit()
 
 
+# retry on operational errors to account for db flakiness with sqlite
+@retry_async_fn(max_attempts=3, retry_on_exceptions=(sa.exc.OperationalError,))
```

Comment:
i love how we have an internal utility with a better UX than our retry settings on tasks 😂 🙃